src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -215,4 +216,16 @@ private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addEventDispatcher(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->variableNode('event_dispatcher')
+                    ->info('The event dispatcher')
```

Comment:
Can you please explain hie I can achieve with it or a use case? I would update the info in a PR afterwards, because the current state is not quite helpful. Many thanks 🙏

Reply:
I all ready updated the info to: 'When given an event dispatcher, twig will fire a pre-render event, just before rendering a template.'
Is that enough?

Reply:
Very nice, thank you 👍